--- rygel-0.20.0/doc/reference/doc-build.am~	2013-05-12 17:15:55.000000000 +0200
+++ rygel-0.20.0/doc/reference/doc-build.am	2013-10-09 19:07:46.751738394 +0200
@@ -77,7 +77,7 @@
 
 endif
 
-RYGEL_GTKDOC_INSTALL_DIR = $(DESTDIR)$(datadir)/gtk-doc/html/lib$(RYGEL_DOCBUILD_MODULE)
+RYGEL_GTKDOC_INSTALL_DIR = $(DESTDIR)$(datadir)/doc/gtk-doc/html/lib$(RYGEL_DOCBUILD_MODULE)
 
 uninstall-hook:
 	rm -rf $(RYGEL_GTKDOC_INSTALL_DIR)
