--- rygel-0.22.0/src/librygel-core/rygel-plugin-information.vala.orig	2014-03-01 15:46:31.000000000 +0100
+++ rygel-0.22.0/src/librygel-core/rygel-plugin-information.vala	2014-04-02 19:40:46.689425216 +0200
@@ -28,11 +28,9 @@
  *
  * A minimal file for the plugin librygel-sompelugin.so looks like this:
  *
- * {{{
  * [Plugin]
  * Name = SomeNameForThePlugin
  * Module = someplugin
- * }}}
  *
  * Name must not contain any whitespaces.
  */
