--- rygel-0.36.0/configure.ac~	2017-11-26 12:15:29.000000000 +0100
+++ rygel-0.36.0/configure.ac	2017-11-26 13:14:02.666952537 +0100
@@ -464,7 +464,7 @@
 AS_IF([test "x$XSLTPROC" != "x"],
       [
        AC_MSG_CHECKING([whether $XSLTPROC can create manpages])
-       $XSLTPROC -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl >/dev/null 2>&1
+       $XSLTPROC -nonet http://docbook.sourceforge.net/release/xsl-ns/current/manpages/docbook.xsl >/dev/null 2>&1
        AS_IF([test "$?" = 0], [AC_MSG_RESULT([yes])], [XSLTPROC=""; AC_MSG_RESULT([no])])
       ])
 AM_CONDITIONAL([HAVE_XSLTPROC], [test x"$XSLTPROC" != x])
